humanoid_league_speaker/src/humanoid_league_speaker/speaker.py

```diff
@@ -82,7 +84,10 @@ def say(self, text, file=False):
         """ Speak this specific text"""
         # todo make volume adjustable, some how like this
         #        command = ("espeak", "-a", self.amplitude, text)
-        command = ("espeak", text)
+        if os.environ['ROBOT_NAME'] in self.female_robots:
```

Comment:
This will raise a KeyError when the `ROBOT_NAME` variable does not exist.
```suggestion
        if os.getenv('ROBOT_NAME') in self.female_robots:
```